misk-redis/src/main/kotlin/misk/redis/Redis.kt

```diff
@@ -226,6 +228,8 @@ interface Redis {
    * another client pushes to it or until timeout is reached. A timeout of zero can be used to block
    * indefinitely.
    *
+   * For Redis Cluster, source and destination must be in the same hash slot.
```

Comment:
What will happen if it's not? will this throw?

Reply:
Yes, updated the docs.